### PR TITLE
SAK-44306: Error on log when trying to upload file with no file selected

### PR DESCRIPTION
**Pull request by FerAlvarez95**

### CODE REVIEW

**Comment by jesusmmp** [comment]
@jonespm FerAlvarez95 signed the cla some months ago.

**Comment by jesusmmp** [comment]
@adrianfish, we have a lot of errors on the log since this one (https://jira.sakaiproject.org/browse/SAK-40906)
was introduced. Please, could you take a look? Thanks!

File: samigo/samigo-app/src/java/com/corejsf/UploadRenderer.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ private WrappedUpload wrapUpload(HttpServletRequest request, String name) {
             Part part = request.getPart(name);
             if (part != null) return new WrappedUpload(part);
         } catch (Exception e) {
-            log.error("Failed to get upload part from request. Null will be returned.", e);
+            log.debug("Failed to get upload part from request. Null will be returned.", e);
```
Comment (ern) [inline samigo/samigo-app/src/java/com/corejsf/UploadRenderer.java:152]: log.warn("Failed to get upload part from request, NULL will be returned, {}", e.toString());

Comment (ern) [inline samigo/samigo-app/src/java/com/corejsf/UploadRenderer.java:152]:
```suggestion
            log.warn("Failed to get upload part from request, NULL will be returned, {}", e.toString());
```

**Review by ern — CHANGES_REQUESTED** [review]

**Pull request merged**